src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -40,8 +40,30 @@
      {@comments(xapi.doc.lines)}
      *
    @end
-   *
    * @@class
+   *
+   * @@param {object=} options - [Configuration object](#/docs).
```

Comment:
`[Configuration object](#/docs)` won't link to anything real, just remove the link.